src/components/transactions/TxListItem/ExpandableTransactionItem.tsx

```diff
@@ -53,4 +53,22 @@ export const ExpandableTransactionItem = ({
   )
 }
 
+export const TransactionSkeleton = () => (
+  <>
+    <Box pt="20px" pb="4px">
+      <Skeleton variant="text" width="35px" />
+    </Box>
```

Comment:
This Skeleton is unnecessary in the Single Tx because there is no label

Reply:
For queued transactions, there's always a label. I guess it's arbitrary if we want to have it or not.